### PR TITLE
README.MD cosmetics

### PR DESCRIPTION
**Pull request by kuraga**
Для людей.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,82 @@
...
+
+For preprocessing, put the decompressed zip file in the main folder and run:
+```bash
 python preprocess.py
 python word2vec.py
 ```
-respectively in code/ . The preprocessed files and trained word embeddings for each domain will be saved in a folder preprocessed_data/.
+respectively in `code/`. The preprocessed files and trained word embeddings for each domain will be saved in a folder `preprocessed_data/`.
+
+You can also find the pre-processed datasets and the pre-trained word embeddings in [[Download]](https://drive.google.com/open?id=1L4LRi3BWoCqJt5h45J2GIAW9eP_zjiNc).
```
Comment (kuraga) [inline README.md:33]: @madrugado BTW, I don't see the "pre-trained word embeddings" there.

Comment (madrugado) [reply README.md:33]: there are no, they could be trained using word2vec.py

Comment (kuraga) [reply README.md:33]: I'm talking about the phrase: You can also find the pre-processed datasets **and the pre-trained word embeddings**.

Comment (kuraga) [reply README.md:33]: @madrugado, I'm talking about the phrase: You **can** also **find** the pre-processed datasets and **the pre-trained word embeddings**. But seems like we can't.

**Pull request merged**